src/lib/middlewares/validateAuthData.unit.spec.js

```diff
@@ -1,23 +1,19 @@
 import { mockRequest, mockResponse } from 'mock-req-res';
-import validateSignupData from './validateSignupData';
+import validateAuthData from './validateAuthData';
 
-describe('#validateSignupData', () => {
+describe('#validateAuthData', () => {
   const validData = {
-    firstName: 'Joe',
-    lastName: 'Doe',
     email: 'joedoe@gmail.com',
     password: 'mypasswod123',
   };
   const invalidData = {
-    firstName: 'Joe',
-    lastName: 'Doe',
     email: 'joedoegmail.com',
     password: 'mypasswod123',
   };
   test('should return no erros if data is valid', () => {
     const next = jest.fn();
```

Comment:
Can we try and use mocks sparingly
e.g. We don't need a mock to confirm that given a wrong email the data will be invalid